mimesis/providers/path.py

```diff
@@ -9,10 +8,10 @@
 class Path(BaseProvider):
     """Class that provides methods and property for generate paths."""
 
-    def __init__(self, *args, **kwargs):
-        # TODO: platform should be a parameter
+    def __init__(self, platform=sys.platform, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.__p = Personal('en')
+        self.platform = platform
```

Comment:
Could you please document all the platforms we support? What happens when user goes with `Path('does-not-exist')`?